circle.yml

```diff
@@ -58,10 +58,10 @@ deployment:
       - cd frontend; yarn dist-staging;
       - ./cg-deploy/deploy.sh public-staging
   trees:
-    branch: sprint-2-development
+    branch: sprint-3-development
```

Comment:
this is right?

Reply:
changed to `sprint-3` so this branch will not deploy.

Reply:
👍